portfolio/src/main/webapp/index.html

```diff
@@ -20,20 +20,30 @@ <h1>Welcome to Rosanna's Portfolio! </h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
 
-        <h2> Week 3 Servlet:</h2>
-        <p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
-        <p>Click here to directly see content requested from /data. It should return a arraylist of the comments inputted:</p>
-        <button onclick="getComment()">Fetch the comment!</button>
-        <div id="content-container"></div>
-
-      
-    <form action="/data" method="POST">
-
-      <p>Enter a comment:</p>
+      <h2> Week 3 Servlet:</h2>
+      <p>Click <a href="/data">here</a> to view content generated by a servlet, /data.</p>
+      <form action="/data" method="POST">
+      <h3>Enter a comment:</h3>
       <!-- passing a parameter called trial comment -->
       <textarea name="text-input">trial comment</textarea>
-      <input type="submit" />
-    </form>
+      <br> 
+      <input type="submit">
+      </form>
+
+      <h3> How many comments do you want to display?</h3>
+      <select name="maxCommentSelected" id="maxCommentSelected">
```

Comment:
If the user doesn't select something here, what's the default behaviour?

Reply:
it would be 3! i shall explain that in the comments